controllers/controllers/workloads/env/vcap_app_builder_test.go

```diff
@@ -38,9 +53,12 @@ var _ = Describe("VCAP_APPLICATION env value builder", func() {
 			Expect(vcapAppValue).To(HaveKeyWithValue("space_name", cfSpace.Spec.DisplayName))
 			Expect(vcapAppValue).To(HaveKeyWithValue("organization_id", cfOrg.Name))
 			Expect(vcapAppValue).To(HaveKeyWithValue("organization_name", cfOrg.Spec.DisplayName))
+			Expect(vcapAppValue).To(HaveKeyWithValue("uris", []any{}))
```

Comment:
gomega offers an idiomatic matcher for empty arrays - `BeEmpty()`. Besides being idiomatic, the matcher outputs a nicer error message on failures. This line could be `Expect(vcapAppValue).To(HaveKeyWithValue("uris", BeEmpty()))`

Same goes for the `Expect` that follows.